nixos/modules/misc/meta.nix

```
@@ -37,7 +37,7 @@ in
         type = listOfMaintainers;
         internal = true;
         default = [];
-        example = [ lib.maintainers.all ];
+        example = literalExpression ''[ lib.maintainers.all ]'';
```

`all` is a weird one. What about `octocat`? Not a real maintainer, but gets the point across with less confusion. Without potentially bothering a real person.

```suggestion
        example = literalExpression ''[ lib.maintainers.octocat ]'';
```

---

it is. we didn't change the existing value here because it seems to be never shown anywhere (the option being internal).

---

heh yeah that's kind of useless.